_data/shirt-dev/shirt-dev.md

```diff
@@ -0,0 +1,6 @@
+---
+name: shirt
+institution: Careerline Tech Center
+quote: Programming is like sex, one mistake and you are providing support for a lifetime.
```

Comment:
We need you to change the quote to fit our CoC

Reply:
sorry, will do

Reply:
#8627